GDJS/Runtime/runtimeobject.ts

```diff
@@ -123,6 +130,16 @@ namespace gdjs {
       for (let i = 0; i < this._behaviors.length; ++i) {
         this._behaviors[i].onCreated();
       }
+
+      const objectEffectsManager = this._runtimeScene
+        .getGame()
+        .getObjectEffectsManager();
+      for (const filterName in this._rendererEffects) {
+        objectEffectsManager.applyEffect(
+          this.getRendererObject(),
+          this._rendererEffects[filterName]
+        );
+      }
```

Comment:
Can this be moved to the constructor? Or at least before the `onCreated` called on behaviors.
But 99% sure this should be in the constructor?

Reply:
Ah right this can't be in the RuntimeObject constructor because this is called before the renderer is created. In this case, still move this before the loop on the behaviors onCreated.
It's possible a behavior would interact with the effects, but they are not applied yet, which could in theory be a problem.

Reply:
Done 👍